### ISSUE TITLE
Turn scheduled charging ON/OFF

### ISSUE THREAD

**Issue by lipigab** [dupondje/nrgkick-ha#7]
Hi,

Is it possible to control the Scheduled charging togle via your integration thus making it available in Home Assistant?

![image](https://github.com/dupondje/nrgkick-ha/assets/44985474/5d3398ac-e393-4ed2-8b19-07396c0057c5)

Thanks,
Gabor

**Comment by dupondje** [comment]
Yes it possible. (Almost) everything you can do in the app can be done via the 'api'. As this is how the app communicates with the device.
It's most likely just setting https://github.com/dupondje/nrgkick-proto/blob/main/nrgcp.proto#L750

Feel free to create a PR :)
